taskcluster/ci/ui-test/kind.yml

```diff
@@ -0,0 +1,67 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+kind-dependencies:
+    - toolchain
+    - build
+
+job-defaults:
+    attributes:
+        build-type: debug
+        code-review: true
+        retrigger: true
+    # dependencies:
```

Comment:
do we still need these commented deps?

Reply:
They'll be needed for the artifact retrieval but I can remove it now since its cleaner. The history will be preserved in this pr.